package.json

```diff
@@ -25,7 +25,9 @@
     "site": "vite build dev --config ./vite.config.js",
     "storybook": "storybook dev -p 6006",
     "test": "vitest run --coverage",
-    "watch": "vitest watch"
+    "watch": "vitest watch",
+    "lint:fix": "eslint . --fix --ignore-path .gitignore",
+    "lint:check": "eslint . --ignore-path .gitignore"
```

Comment:
Do we also need to add this `--ext js,cjs,mjs` to `lint:fix` and `lint:check`? :)

Reply:
It looks like we are doing that in elements.
Also `lint:fix` could be just `lint`

Reply:
I don't think we need the extension flag here, because there are no .cjs or .mjs files that we need eslint to look at, like we do in @warp-ds/elements. And we are using `lint:check` in the github action, therefore the differentiation between fix and check scripts. We might change that when adding Husky tool to this repo, which would run a linting scripts before committing or pushing code.

Reply:
Updated in [189a8b4](https://github.com/warp-ds/vue/pull/152/commits/189a8b40b2efcea70a1d8857607cfc58db6b687d)